fe2-android/app/build.gradle

```
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
+        // As it's purpose is to clear physical data between each tests, its is not used yet.
```

`its is`?

---

the 'clearPackage: true'

---

I don't really see how to improve the comments, but I want a comment to explain its purpose and another to explain why it is disabled

---

I should have been clearer. `its is` looks like a typo 😄 
```suggestion
        // Its purpose is to clear physical data between each tests, but it is not used yet.
```

---

And indeed the comment is clear.